### ISSUE TITLE
Upgrading from 32 to 64 bit went poorly (bad uninstall)

### ISSUE THREAD

**Issue by karlmortensen** [sqlitebrowser/sqlitebrowser#274]
Running Windows 7, 64-bit OS.

I had a 32-bit version of 3.5.1. This time around I decided to go 64-bit for 3.6.0v2. I uninstalled the 32-bit version (which wasn't registered in Windows 7's Add/Remove Programs list, so I had to find it and use uninstall.exe), but unfortunately it didn't get everything. 

I installed 64-bit, and double clicked on a .db file. It brought up the 32-bit version that I thought I had just uninstalled. I did this a couple times and then uninstalled 64-bit, searched my hard drive manually deleting every sqlitebrowser.exe I could find.  I re-installed 64-bit, but now I could not double click on my .db files and make them open in sqlite browser. It associated with the next program on the list (just happened to be 7zip, and 7zip choked...). 

In any case, digging into the registry, I found that the key HKEY_CLASSES_ROOT\Applications\sqlitebrowser.exe\shell\open\command\Default was still set to  "C:\Program Files **(x86)**\SqliteBrowser3\bin\sqlitebrowser.exe" "%1". This confused Windows 7 and would not let it associate the new, 64-bit install with .db. Once I whacked that and re-installed 64-bit, it was shiny.

tl;dr: Make sure that the uninstall whacks that reg key so an install with a different target (32/64/ia/etc) will not confuse Windows.

Other than that, great job with 3.6.0v2. I like it. :-)

**Comment by rp-** [comment]
our installer doesn't set any file associations

**Comment by karlmortensen** [comment]
Ah. That's why it didn't unset them. :-)
My bad.
